### ISSUE TITLE
Atata error with NUnit3TestAdapter 3.15.0

### ISSUE THREAD

**Issue by hamiltonmarques** [atata-framework/atata#301]
Hi!
I updated the NUnit3TestAdapter to latest version (3.15.0) with NuGet in my project and now Atata throws this error:
![image](https://user-images.githubusercontent.com/23129723/63610421-d3826e80-c5af-11e9-8dab-c6f955f067bb.png)
When I back to previous version (3.14.0) the tests run fine.
Could you check if it's happening with you too, please?

**Comment by YevgeniyShunevych** [comment]
Hey,

The same for me for the `OneTimeSetUp` method located in `SetUpFixture` that doesn't execute. In your case, I guess `AtataContext` initialization is located there. Example:

```cs
[SetUpFixture]
public class SetUpFixture
{
    [OneTimeSetUp]
    public void GlobalSetUp()
    {
        // This doesn't get executed.
    }
}
```

It seems like a bug of NUnit3TestAdapter v3.15.0. Will have to take a look deeper.

For now I highly recommend not to update NUnit3TestAdapter package to v3.15.0, but use v3.14.0.

Thanks for reporting this issue.

**Comment by YevgeniyShunevych** [comment]
Hey,

It seems like recently released NUnit3TestAdapter v3.15.1 got a fix. Works well for me. You can verify this version with your solution.

**Comment by hamiltonmarques** [comment]
Hi,
NUnit3TestAdapter v3.15.1 works well for me too.
Thanks!
